webpack.config.js

```diff
@@ -3,14 +3,20 @@
 const path = require("path");
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 
-/** @type {import("webpack").Configuration} */
-module.exports = {
+/**
+ * webpack configuration
+ * @param {import("webpack").ConfigurationFactory} env
+ * @param {import("webpack").ConfigurationFactory} argv
```

Comment:
I don't believe `ConfigurationFactory` is exported by webpack officially. Also, webpack-cli is responsible for these types. https://github.com/webpack/webpack/issues/11630#issuecomment-838510078

```suggestion
 * @param {Record<string, any>} env
 * @param {Record<string, any>} argv
```

Reply:
Done :)